desktop/apps/consignments2/components/describe_work_desktop/index.js

```diff
@@ -109,18 +113,21 @@ function DescribeWorkDesktop (props) {
             <Field name='height' component={renderTextInput}
               item={'height'}
               label={'Height*'}
+              warn={numberWarning}
```

Comment:
Oh man. I feel like I wrote most of it before discovering redux-form, but it's so so so nice to not have to add new state just to do things like this 😁